bubblegum/program/src/lib.rs

```diff
@@ -850,6 +848,12 @@ pub mod bubblegum {
             BubblegumError::TreeAuthorityIncorrect,
         );
 
+        if !authority.contains_mint_capacity(1) {
```

Comment:
The `spl-compression` program also does some checks during appends, and will return an error if a tree is full (for example, when eventually reaching [this](https://github.com/solana-labs/solana-program-library/blob/6bcd051d384f4ebcf8d6933291d8a3eacef412bf/libraries/concurrent-merkle-tree/src/concurrent_merkle_tree.rs#L214) line. Was just wondering, do we need to do the parallel accounting? (or, for example, are there circumstances where the max capacity would be defined as less than the theoretical max for a tree?)